### ISSUE TITLE
Exposing metrics endpoints while using multiple fluentd workers

### ISSUE THREAD

**Issue by kobejn-jb** [kube-logging/logging-operator#1498]
**Is your feature request related to a problem? Please describe.**
I'm musing multiple fluentd workers in each fluentd pod, I need this to handle huge amount of logs from on my GKE cluster with reduced number of fluentd pods (specially on startup statefulset is slow to scale up and few fluentd pods can't handle all the data from fluent-bit daemonset that starts very fast)

**Describe the solution you'd like**
In case of using multiple workers `fluentd.workers: X` the operator should expose ports for metrics endpoints in fluentd pods, pseudo code example:
```
(loop from zero to worker count -1)
    - containerPort: 24231 + <worker_id>
      name: monitor-<worker_id>
      protocol: TCP
```

Also fluentd monitor service and serviceMonitor should account for those ports

**Describe alternatives you've considered**
I could't think of any alternative solutions, I guess users can create their own service and service monitor but ports would have to be exposed in the pod

**EDIT:**
There actually is a way to handle this, fluentd already exposes aggregated_metrics endpoint, for now this setting:
```
  fluentd:
    metrics:
      path: aggregated_metrics
      serviceMonitor: true
```

The only thing I do not understand is why standard `/metrics` endpoint is not available in fluentd pod anymore after applying this settings, should it not just update `path` in serviceMonitor? Just curious, it's not a major issue.

**Additional context**
For now when using multiple workers only metrics from worker 0 are exposed:
![image](https://github.com/kube-logging/logging-operator/assets/60479855/6c67e6a0-ad4b-4710-aef3-b24500ed7351)

**Comment by aslafy-z** [comment]
Hi,
I believe this is already solved in https://github.com/kube-logging/logging-operator/pull/1417 and will be released with version 4.4.0.

**Comment by kobejn-jb** [comment]
Oh, nice. Thank you :+1:
